changelog.d/gh-6001.fixed

```diff
@@ -0,0 +1 @@
+Resolve imported types for typed metavariables
```

Comment:
I think this changelog entry would be a bit cryptic for anybody (even from PA) not involved in fixing this issue.